tests/unit/beacons/test_inotify.py

```diff
@@ -39,10 +39,33 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.tmpdir, ignore_errors=True)
 
+    def test_non_list_config(self):
+        config = {}
+
+        ret = inotify.validate(config)
+
+        self.assertEqual(ret, (False, 'Configuration for inotify beacon must'
+                                      ' be a list.'))
+
     def test_empty_config(self):
         config = [{}]
-        ret = inotify.beacon(config)
-        self.assertEqual(ret, [])
+        ret = inotify.validate(config)
+        _expected = (False, 'Configuration for inotify beacon must include files.')
+        self.assertEqual(ret, _expected)
+
+    def test_files_none_config(self):
+        config = [{'files': None}]
```

Comment:
I won't say same as above here, although we don't actually use `config` anywhere else.
But for the sake of readability, let's leave it.
The same for the next test.